components/EmotionDetails.js

```diff
@@ -1,19 +1,15 @@
+import EmotionCard from "./EmotionCard";
 import Heading from "./Heading";
 import styled from "styled-components";
 
-export default function EmotionDetails({ emotion, onBack }) {
+export default function EmotionDetails({ entry, onBack }) {
   return (
     <>
       <Heading>Emotion Details</Heading>
       <StyledButton type="button" aria-label="navigate-back" onClick={onBack}>
         ←
       </StyledButton>
```

Comment:
You can work with an href here:

```suggestion
      <StyledLink aria-label="navigate-back" href="/">
        ←
      </StyledLink>
```

You most likely want to keep the same styled and instead of 
const StyledButton = styled.button
You can write:
const StyledLink = styled(Link)

Don't forget to import the Link component from Next